### ISSUE TITLE
Backcompat support for V2 PCL for V3 PCL with dotnet TFM

### ISSUE THREAD

**Issue by yishaigalatzer** [NuGet/Home#922]
This is a place holder for a new feature in project.json allowing specifying a fallback TFM if the framework is `dotnet`

Although there is no guarantee of the fallback TFM to actually work, it will likely work in most cases so we are going to enable it by user explicit choice only for project to project references.

Potential shape (specific suggestion is being worked on).

```json
"framworks" : {
    "dotnet" : {
         "allowAlso" : "portable-X+Y+Z"
    }
}
```

This will also require a change in the project system by the managed languages team

**Comment by emgarten** [comment]
Fixed NuGet/NuGet3@a5bc907d36ddaa8d4fa6c499e50d7ebf8993ed39

**Comment by davidfowl** [comment]
Can we call it allow
